docs/topics/testing/tools.txt

```diff
@@ -1590,9 +1590,16 @@ your test suite.
     .. deprecated:: 4.1
 
         Support for passing a response object and a formset name to
-        ``assertFormsetError()`` is deprecated and will be removed in Django
+        ``assertFormSetError()`` is deprecated and will be removed in Django
         5.0. Use the formset instance directly instead.
 
+.. method:: SimpleTestCase.assertFormsetError(formset, form_index, field, errors, msg_prefix='')
+
+    .. deprecated:: 4.2
+
+        This old spelling of ``assertFormSetError()`` is deprecated and will be
+        removed in Django 5.1.
+
```

Comment:
I'd add `deprecated:: 4.2` annotation to the `assertFormSetError` instead of documenting a deprecated method.

Reply:
Fixed in https://github.com/django/django/pull/16110/commits/54fed788d0dd9fedc6d5c83a528b3dfff90c0c09.